projects/1.19/assets/serene-seasons/sereneseasons/lang/zh_cn.json

```
@@ -0,0 +1,38 @@
+{
+  "commands.sereneseasons.usage": "/season <get/set> [参数]",
+  "commands.sereneseasons.getseason.success": "当前的季节为%s, 第%s/%s天, 第%s/%s刻",
+  "commands.sereneseasons.setseason.success": "季节已设定为%s",
+  "commands.sereneseasons.setseason.fail": "季节%s无效",
+  "commands.sereneseasons.setseason.disabled": "季节更替已被禁用！",
+
+  "itemGroup.tabSereneSeasons": "Serene Seasons",
+
+  "block.sereneseasons.season_sensor": "季节指示器",
+
+  "item.sereneseasons.ss_icon": "静谧四季图标",
+  "item.sereneseasons.calendar": "日历",
+
+  "desc.sereneseasons.fertile_seasons": "丰收时刻",
```

原文：**Fertile Seasons**，这里应该是一个“时段”，`时刻`在这里就有些不妥了

---

我其实想的是它有一种类似于“尖峰时刻”那样的感觉，所以我也采用了时刻这种在一段时段里有一个很短的时间周期这样的意思，用来表示丰收的时段只有短短一个季节，你觉得呢

---

其持续时间为多久？在游戏中如何显示，效果是什么？
`丰收季`、`丰收日`、`农忙季` 等是不是更恰当？

---

没在游戏中见到过也没在wiki中找到...
我刚才看了一眼上一个版本，使用的是
```suggestion
  "desc.sereneseasons.fertile_seasons": "收获季节",
```
我决定用这个了，你们觉得呢

---

看了一眼之前公开的源码，这个是在配置中设置过才会出现的选项
每个世界的收获季节由种子决定，启用农作物受季节影响后就会受其影响，启用显示收获季节后就会显示这个desc

---
顺带一提这里的所有配置项和格式全都是硬编码（）

---

那我觉得`收获季节`应该可以